python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -745,12 +763,65 @@ def evaluate_tick(self, context: "ScheduleEvaluationContext") -> ScheduleExecuti
             skip_message = None
 
         # clone all the run requests with the required schedule tags
-        run_requests_with_schedule_tags = [
-            request.with_replaced_attrs(
-                tags=merge_dicts(request.tags, DagsterRun.tags_for_schedule(self))
+        run_requests_with_schedule_tags = []
```

Comment:
`resolved_run_requests` would be a clearer name now that we're doing more than just adding schedule tags